ftest.sh

```diff
@@ -127,6 +128,17 @@ wq
 EOF
     sudo mount /mnt/daos
 fi
+
+# make sure to set up for daos_agent
+current_username=\$(whoami)
```

Comment:
Just a suggestion, but could probably just use `$USER` or `$LOGNAME` and not depend on `whoami` here in which case you'd just use it below directly in place of assigning to `current_username`.

Reply:
I went with whoami instead of $USER out of habit, I generally see $USER advised against because it can change/be unset. In this context I don't think it's a problem to use $USER but whoami works alright.

Reply:
Yeah, fair enough.  Was more just an FYI.